README.md

```diff
@@ -14,7 +14,20 @@ For further information about the model there are useful vignettes available by
 browseVignettes("bw")
 ```
 
-Other packages required are listed in the `requirements.txt` file.
+After having installed the `bw` package run 
+
+```
+requirements.R
```

Comment:
Make it a bit clearer here that this is opening and running the file as its in a code block it could be confused with a line of code to run (my confusion :) )